README.md

```diff
@@ -32,7 +32,7 @@ Building Scylla with the frozen toolchain `dbuild` is as easy as:
 ```bash
 $ git submodule update --init --force --recursive
 $ ./tools/toolchain/dbuild ./configure.py
-$ ./tools/toolchain/dbuild ninja build/release/scylla
+$ ./tools/toolchain/dbuild ninja -C build scylla:RelWithDebInfo
```

Comment:
Please confirm this actually works with cmake now (i.e., cmake is included in the container image).

Reply:
... the syntax looks OK to me, otherwise.

Reply:
totally tangential side comment: lower down in this file, we have a mention of `test.py`, with a pointer to `docs/dev/testing.md`. That document seems to say nothing about `--no-gather-metrics`, or `systemd-run` :( so I think the currently documented way will encounter cgroups permission errors. Oh well.

Reply:
> Please confirm this actually works with cmake now (i.e., cmake is included in the container image).

we run `install-dependencies.sh` when building the toolchain image, see https://github.com/scylladb/scylladb/blob/298cafff3537950f1b17b1fcf119e3d831bcd786/tools/toolchain/Dockerfile#L11 . so it should work.

Reply:
> totally tangential side comment: lower down in this file, we have a mention of `test.py`, with a pointer to `docs/dev/testing.md`. That document seems to say nothing about `--no-gather-metrics`, or `systemd-run` :( so I think the currently documented way will encounter cgroups permission errors. Oh well.

@xtrey hello Andrei, do you think we should improve the help message of `--gather-metrics` and related documentation?

Reply:
@tchaikov we can improve it. But there are no issues with metrics inside the toolchain.